pyscriptjs/src/plugin.ts

```diff
@@ -74,11 +74,23 @@ export class PluginManager {
     }
 
     beforeLaunch(config: AppConfig) {
-        for (const p of this._plugins) p.beforeLaunch(config);
+        for (const p of this._plugins) {
+            try{
+                p.beforeLaunch(config);
+            } catch(e) {
+                logger.error('Error while calling beforeLaunch hook', e);
+            }
+        }
     }
 
     afterSetup(runtime: Runtime) {
-        for (const p of this._plugins) p.afterSetup(runtime);
+        for (const p of this._plugins) {
+            try {
+                p.afterSetup(runtime);
+            } catch(e) {
+                logger.error('Error while calling afterSetup hook', e);
```

Comment:
As above - is it possible to add a reference to the specific plugin here? "Error while calling afterSetup hook of plugin 'p'" or something?